README.md

```diff
@@ -163,6 +163,10 @@ Inline styles object to be passed to the image wrapper element.
 
 Inline styles object to be passed to the image element.
 
+#### useNaturalImageDimensions (optional)
+
+If true then the pixel crop is calcaulted using natural dimensions instead of the default ones
```

Comment:
Please change to:

"If true then the pixel crop is calculated using natural dimensions instead of the rendered dimensions (defaults to true)."